src/ConfigContext.ts

```diff
@@ -32,9 +32,9 @@ let configContext: Readonly<ConfigContext> = {
   platform: Platform.Portal,
   allowedParentFrameOrigins: [
     `^https:\\/\\/cosmos.azure.(com|cn|us)$`,
-    `^https:\\/\\/[\\.\\w]+.portal.azure.(com|cn|us)$`,
-    `^https:\\/\\/[\\.\\w]+.ext.azure.(com|cn|us)$`,
-    `^https:\\/\\/[\\.\\w]+microsoftazure.de$`
+    `^https:\\/\\/[\\.\\w]*portal.azure.(com|cn|us)$`,
+    `^https:\\/\\/[\\.\\w]*ext.azure.(com|cn|us)$`,
+    `^https:\\/\\/[\\.\\w]*microsoftazure.de$`
```

Comment:
please test with malicious.germanycentral.cloudapp.microsoftazure.de

Reply:
I'd vote we use `^https:\\/\\/[\\.\\w]*ext.microsoftazure.de$` so only portal extensions are whitelisted. The one above whitelists any public azure resource in blackforest.

Reply:
also add a test for the below (I know you regex works, but to avoid regressions by common mistakes)

maliciousazure.com
maliciousportalsazure.com

Reply:
My original regex would have allowed the second one :/

I've added the above cases and explicitly whitelisted `https://cosmos-db-dataexplorer-germanycentral.azurewebsites.de/` since it is what we use in German cloud